recipes/acado/all/conanfile.py

```diff
@@ -59,6 +59,8 @@ def _configure_cmake(self):
         self._cmake.definitions["ACADO_INTERNAL"] = False
         self._cmake.definitions["ACADO_BUILD_CGT_ONLY"] = self.options.codegen_only
 
+        self._cmake.definitions["CMAKE_CXX_FLAGS"] = "-w"
```

Comment:
`-w` is a gcc flag. What about MSVC and other compilers?

Reply:
hmpf, need to google that :-)

Reply:
![image](https://user-images.githubusercontent.com/23190927/103696103-145d0680-4f9e-11eb-9b83-62d9643e0646.png)

https://stackoverflow.com/a/65287343/11879045

Reply:
according to https://docs.microsoft.com/de-de/cpp/build/reference/compiler-option-warning-level?view=msvc-160 it would be `/w` for msvc.

Reply:
I would do it like in this recipe https://github.com/conan-io/conan-center-index/blob/9cee683f1cf63e9b46a8992867db672b43bd14e4/recipes/harfbuzz/all/conanfile.py#L74

Reply:
Is this ok as well?
```python
        if self.settings.compiler == "Visual Studio":
            self._cmake.definitions["CMAKE_C_FLAGS"] = "/w"
            self._cmake.definitions["CMAKE_CXX_FLAGS"] = "/w"
        else:
            self._cmake.definitions["CMAKE_C_FLAGS"] = "-w"
            self._cmake.definitions["CMAKE_CXX_FLAGS"] = "-w"
```

Reply:
if this silences all warnings, why don't you just put it into a profile, than it will apply to all recipes

Reply:
From personal experience, the MSVC tools also accept arguments prefixed with `-`.
Perhaps this is undocumented behavior?

Reply:
> if this silences all warnings, why don't you just put it into a profile, than it will apply to all recipes

This one is special, as it throws out 170.000 lines of warnings for a 2 second compile.

How about a conan option to enable silent mode?

Reply:
> > From personal experience, the MSVC tools also accept arguments prefixed with `-`.
> > Perhaps this is undocumented behavior?
> 
> Then it would be the one-liner again

It didn't complain in https://github.com/conan-io/conan-center-index/pull/4061#issuecomment-754870043

Reply:
Ok, then rollback

Reply:
My only comment is to add a comment to why you're doing this (for documenting purposes).

Reply:
will do